core/src/main/res/drawable/ic_open_in_new_24dp.xml

```diff
@@ -16,8 +16,10 @@
   ~
   -->
 
-<vector android:height="24dp"
+<vector xmlns:tools="http://schemas.android.com/tools"
+  android:height="24dp"
     android:viewportHeight="24" android:viewportWidth="24"
     android:width="24dp" xmlns:android="http://schemas.android.com/apk/res/android">
-    <path android:fillColor="#565656" android:pathData="M19,19H5V5h7V3H5c-1.11,0 -2,0.9 -2,2v14c0,1.1 0.89,2 2,2h14c1.1,0 2,-0.9 2,-2v-7h-2v7zM14,3v2h3.59l-9.83,9.83 1.41,1.41L19,6.41V10h2V3h-7z"/>
+    <path android:fillColor="?colorOnSurface" android:pathData="M19,19H5V5h7V3H5c-1.11,0 -2,0.9 -2,2v14c0,1.1 0.89,2 2,2h14c1.1,0 2,-0.9 2,-2v-7h-2v7zM14,3v2h3.59l-9.83,9.83 1.41,1.41L19,6.41V10h2V3h-7z"
```

Comment:
I remember us having issues using attributes in vectors, the solution will probably be to add a resource to the night drawables